docs/how-to-guides/develop-a-tool/use-file-path-as-tool-input.md

```diff
@@ -0,0 +1,98 @@
+# Use FilePath as tool input
+
+In this document, we will guide you through the detailed process of how to use FilePath as tool input with 'hello-world' tool example.
+
+## Prerequisites
+
+Since FilePath is still in test currently, you need to install private version SDK and Extension.
+
+1. Download private version promptflow SDK
+
+    ```bash
+    pip install "promptflow==0.0.106113823" --extra-index-url https://azuremlsdktestpypi.azureedge.net/test-promptflow/
+    ```
+
+2. Download this [VSCode Extension](https://aka.ms/pfvsctest), install from VSIX:
```

Comment:
For install promptflow extension, better to keep consistent with create-and-use-tool-package.md:
![image](https://github.com/microsoft/promptflow/assets/49388944/8bc064e8-7cce-42ec-9f46-857d3e091b69) 
and use this link : https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow since it is more official.